checks.d/process.py

```diff
@@ -305,6 +309,13 @@ def check(self, instance):
         pid = instance.get('pid')
         pid_file = instance.get('pid_file', None)
 
+        if self._conflicting_procfs:
+            self.warning('The `procfs_path` defined in `process.yaml` is different from the one defined in '
+                         '`datadog.conf`. This is currently not supported by the Agent')
```

Comment:
We should probably state here we're defaulting to the `datadog.conf` specified path.